### ISSUE TITLE
Javascript Dead Reckoning

### ISSUE THREAD

**Issue by cbrittingham** [open-dis/open-dis-javascript#7]
Not really an issue, just a question...

Is there any chance I am overlooking a javascript implementation of the DIS Dead Reckoning algorithms?  

I found the Java version (DeadReckoner.java) here:
[https://github.com/open-dis/open-dis-java/blob/09f96967e7e25e86d61420713e1f2f429c94cf80/src/main/java/edu/nps/moves/deadreckoning/DeadReckoner.java](url)

**Comment by leif81** [comment]
You're right there isn't a JavaScript implementation for dead reckoning yet.

Pull requests are more than welcome!

**Comment by cbrittingham** [comment]
Thanks.  Just wanted to verify before I reinvented the wheel.

**Comment by Skhmt** [comment]
Did you succeed?
